app/interactors/send_team_hours_update.rb

```diff
@@ -0,0 +1,15 @@
+class SendTeamHoursUpdate
+  include Interactor
+  include HasHarvest
+
+  def call
+    Team.active.each do |team|
+      from = Date.this_week.first
+      to = Date.this_week.last
```

Comment:
Ah, yes, I get you. Do you think a `range` is good enough? or a two-param `from`, `to` setup?